### DISCUSSION TITLE
CrazyRadio

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#909] [Q&A]
Hi, If I use the one crazyflie radio 2.0 in client to read about the UWB information and other console. Can I use another radio 2.0 insert other laptop to run the python script ? Does it work successful ? Thanks.

**Comment by ataffanel** [comment]
Hi,

My understanding is that you want to establish two connections from two different computer/radio to the same Crazyflie.

Unfortunately, this is not currently possible: there can be only one connection active at a time to one Crazyflie. I see 2 solutions: you can either write your script to run in the client, the client is designed in such a way that adding "tabs" is fairly modular. So you can add your own tab to run your script. Another solution is to implement in your script the functionalities you need from the client, things like logging with real-time display can be implemented in python.

**Reply by CalvinKoung** [reply]
Hi, I can understand. But if I use the client calibrate the UWB anchor. And I want to run the script in my laptop (because my laptop is mac m1 chip it seems like no compare with client configuration). So this will led to somthing wrong ? (Two different computer one is calibrate anchor and one run the python script. )

And I try to follow the toturial add logging in the python script, but enconter the error issues.

**Reply by ataffanel** [reply]
Hi, as long as you do not connect from both side at the same time this will not cause any problems: the UWB settings will be written in the anchors and so you can disconnect the client and connect your scripts to use the system.

**Reply by CalvinKoung** [reply]
@ataffanel  Ok, I got it. Thanks

**Reply by amitx64** [reply]
@ataffanel Hi, I designed crazyradio as per your schematic. But, when I tried to go in DFU mode for uploading Crazyradio2-bootloader it is not going in dfu mode and even that I am not able to connect crazyradio with stlink. what shoud I do? Any suggestion?

**Reply by ataffanel** [reply]
@amitx64 This sounds like a completely new topic, can you please create a new question about it.
